src/lib/constants.ts

```diff
@@ -88,6 +88,7 @@ export const enum Tasks {
 	FishingActivity = 'fishingActivity',
 	MiningActivity = 'miningActivity',
 	SmithingActivity = 'smithingActivity',
+	SmithedActivity = 'smithedActivity',
```

Comment:
the names Smithed  and SmithedActivity just make no sense, you should rename  the existing one to smelting, and the new one to Smithing (And test that change). I'll lock smithing for a couple hours to ensure it doesnt cause any issue when i release it.

Reply:
you'll also need to update anything else relevant that needs to be, so everything smelting related is named appropriately and so on